packages/graphql/src/schema/get-nodes.ts

```diff
@@ -186,6 +186,33 @@ function getNodes(definitionNodes: DefinitionNodes, options: { callbacks?: Neo4j
             cartesianPointInTypeDefs = nodeFields.pointFields.some((field) => field.typeMeta.name === "CartesianPoint");
         }
 
+        const globalIdFields = nodeFields.primitiveFields.filter((field) => field.isGlobalIdField);
+
+        if (globalIdFields.length > 1) {
+            throw new Error(
+                "Only one field may be decorated with an '@id' directive where the global argument is set to true"
```

Comment:
I'm not 100% sure, but maybe this is slightly clearer?
```suggestion
                "Only one field may be decorated with an '@id' directive when the global argument is set to true"
```

Reply:
```suggestion
                "Only one field may be decorated with an '@id' directive with the global argument set to `true`"
```